### PR TITLE
conntrack e2e test debug connections

### PR DESCRIPTION
**Pull request by aojea**
adding more debug to the conntrack tests, it turns out some jobs like KIND are not logging the IP addresses of services, pods and endpoints so the conntrack table dump is useless :shrug: 

xref #91236

```release-note
NONE
```

### CODE REVIEW

**Comment by aojea** [comment]
/assign @wojtek-t 
/hold
only just to see the logs and check that all the info needed is there,

**Comment by wojtek-t** [comment]
/lgtm
/approve

@aojea - please cancel hold when you will be ready

File: test/e2e/network/conntrack.go [hunk 1]
```diff
@@ -273,6 +301,7 @@ var _ = SIGDescribe("Conntrack", func() {
 			framework.ExpectNoError(err)
 			framework.Logf("Pod client logs: %s", logs)
 			framework.Failf("Failed to connect to backend 2")
+
```
Comment (wojtek-t) [inline test/e2e/network/conntrack.go:304]: nit: remove this empty line if you're going to submit

Comment (aojea) [reply test/e2e/network/conntrack.go:304]: done

**Comment by aojea** [comment]
pull-kubernetes-e2e-kind-ipv6 — Job failed.

has a conntrack failure, endpoints are published but there is not traffic from the client pod to the new endpoint
We should verify that the iptables rules are there, so the client-pod traffic is DNATed to the new pod
/hold

**Comment by aojea** [comment]
test is passing but I don't fully understand the logic with the NodePort in GCE, the nat reply dst is 10.64.1.1:18222 :thinking: 
That's weird, in KIND jobs is replying to the original IP, is like it is double natting or something

```
grep 10.64.3.190 build-log.txt | grep -v Jul
I0701 11:14:47.120] udp      17 26 src=10.64.3.190 dst=10.40.0.3 sport=12345 dport=31361 [UNREPLIED] src=10.40.0.3 dst=10.64.3.190 sport=31361 dport=12345 mark=0 use=1
I0701 11:14:47.136] udp      17 29 src=10.64.3.190 dst=10.40.0.3 sport=12345 dport=31361 [UNREPLIED] src=10.40.0.3 dst=10.64.3.190 sport=31361 dport=12345 mark=0 use=1
I0701 11:14:47.142] udp      17 27 src=10.64.3.190 dst=10.40.0.3 sport=12345 dport=31361 src=10.64.1.227 dst=10.64.1.1 sport=80 dport=18222 mark=0 use=1
```

**Comment by aojea** [comment]
now that we fixed the iptables lock problem we can start debugging this again
/test all

**Comment by aojea** [comment]
/retest
endpoints tests failures :eyes:

**Comment by aojea** [comment]
sig-cli failures
/retest

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by aojea** [comment]
/retest
[sig-storage] PersistentVolumes-local [Volume type: dir-link] Two pods mounting a local volume one after the other should be able

**Pull request merged**